process/interceptors/factory/shardInterceptedDataFactory_test.go

```diff
@@ -234,6 +234,23 @@ func TestShardInterceptedDataFactory_CreateInterceptedShardHdrShouldWork(t *test
 	assert.True(t, ok)
 }
 
+func TestShardInterceptedDataFactory_CreateInterceptedMetaHdrShouldWork(t *testing.T) {
+	t.Parallel()
+
+	marshalizer := &mock.MarshalizerMock{}
+	emptyHdr := &block.Header{}
+	emptyHdrBuff, _ := marshalizer.Marshal(emptyHdr)
+
+	midf, _ := factory.NewShardInterceptedDataFactory(createMockShardArgument(), factory.InterceptedMetaHeader)
+
```

Comment:
remove empty line.

Reply:
Removed or added on the whole test files as to conform to AAA